.github/docker/centreon-web/alma8/slim-configuration/exec.txt

```diff
@@ -22,4 +22,7 @@ service mysql stop
 
 rm -f /tmp/gorgone/*.ipc /var/log/centreon*/*.log /var/lib/centreon-gorgone/history.sdb
 
+dnf install -y vim
```

Comment:
is vim mandatory (it adds some weight to the container if not)

Reply:
this was mainly for testing purpose in order to allow slimtoolkit to detect binary usage